package.json

```diff
@@ -42,7 +42,7 @@
       },
       "brave-core": {
         "dir": "src/brave",
-        "branch": "master",
+        "branch": "disable_field_trials",
```

Comment:
does this need to be here?

Reply:
will be reverted once PR builder goes through